src/chip/checkbox-chip.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let inputClass = null;
+  export let chipClass = null;
+
+  export let checked = false;
+  export let value;
+  export let disabled = false;
+  export let title = null;
+
+  export let small = false;
+  export let outline = false;
+  export let noPadding = false;
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<label class={classes('chip-wrapper', _class)}>
+  <input
+    bind:checked
+    {value}
+    type="checkbox"
+    class={inputClass}
+    {disabled}
+    on:change={(e) => dispatch('change', { value: e.target.value, checked: e.target.checked })}
```

Comment:
Then I suppose we had a misunderstanding. All events emitted from components are actually custom, with only relevant properties at quick access.

Reply:
ooh, that was a lengthy discussion). I don't see on what basis did you decide that these are the only "relevant" properties, while in the same time wanting to use an array of event listeners to give full power and flexibility to the user and not decide by yourself the relevant events.
I believe that giving the full event object will be most flexible as there are often other properties needed for better control (for example, preventing default). It would be better if the native event is augmented with the relevant properties for quick access
```suggestion
    on:change={(e) => dispatch('change', Object.assign(e, { value: e.target.value, checked: e.target.checked }))}
```
or just sent along with them
```suggestion
    on:change={(e) => dispatch('change', { event: e, value: e.target.value, checked: e.target.checked })}
```

Reply:
This is reasonable. I like the second option more. But the same issue then needs to be fixed in all components, so I'll create an issue for this and make it a separate PR.